octoprint_mrbeam/__init__.py

```diff
@@ -762,6 +762,8 @@ def on_ui_render(self, now, request, render_kwargs):
         # template, using the render_kwargs as provided by OctoPrint
         from flask import make_response, render_template, g
 
+        sockjs_connect_timeout = settings().getInt(["devel", "sockJsConnectTimeout"])
```

Comment:
what is devel?

Reply:
devel is on off the octoprint config settings

Reply:
ok but I understand it shouldn't be called devel then right? no need to do it here but we could create a tech debt for it or a meeting to discuss it :)